apps/condo/domains/news/schema/NewsItemRecipientsExportTask.test.js

```diff
@@ -113,12 +113,15 @@ describe('NewsItemRecipientsExportTask', () => {
             test('admin can', async () => {
                 const [objCreated] = await createTestNewsItemRecipientsExportTask(adminClient, adminClient.user, dummyO10n)
 
-                const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id)
-
+                const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id, {
+                    status: COMPLETED,
+                })
+                
                 expect(obj.dv).toEqual(1)
                 expect(obj.sender).toEqual(attrs.sender)
                 expect(obj.v).toEqual(2)
```

Comment:
Looks like this expectation will fail in the case of the worker being done before the updating within the test.